content/en/apps/guides/forms/rdtoolkit-integration.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Rapid Diagnostic Toolkit Integration"
+linkTitle: "RDToolkit Integration"
+weight:
+description: >
+  Integration for the Rapid Diagnostic Toolkit (RDToolkit)
+keywords: RDToolkit RDT
+
+---
+
+_Introduced in v3.12.0_
+
+CHT has an integration with [Rapid Diagnostic Toolkit (RDToolkit)](https://github.com/dimagi/rd-toolkit) which is an Android app developed by Dimagi that facilitates the use of Rapid Diagnostic Test (RDT) in global health care. 
+The integration is only available in the CHT Android app and consists of two Enketo widgets (rdtoolkit-provision and rdtoolkit-capture) that are used in XForms to provision RDT and capture results.
+
+## Create the XForms
+It is required to create one form for provisioning RDT and one form for capturing RDT results, see [the documentation](https://docs.communityhealthtoolkit.org/apps/tutorials/app-forms/) for more details about app forms.
+
+### Provisioning RDT form
+The following labels are required in the “appearance” tag:
+* `rdtoolkit-provision`: To attach the Enketo widget that connects with the RDToolkit app.
+* `rdtoolkit-action-btn`: To attach the action button that provisions RDT, this button will open the RDToolkit app.
+
+This form has the following mandatory fields:
```

Comment:
It would be helpful to specify where those fields are expected to be. Are they are the top level, or adjacent to the rdtoolkit field?

Reply:
Good catch! So, I've included the image of the xls, but definitely adding the expected level will be better.
This is fixed!